app/views/ucas_contacts/show.html.erb

```diff
@@ -170,6 +170,25 @@
     <h2 class="govuk-heading-l">Email alerts for new applications</h2>
 
     <dl class="govuk-summary-list govuk-!-margin-top-6 govuk-!-margin-bottom-9">
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">
+          Get alerts from UCAS?
+        </dt>
+        <dd class="govuk-summary-list__value">
+          <p class="govuk-body" data-qa="provider__send_application_alerts">
+            <% if @provider.send_application_alerts.present? %>
+              <%= t("edit_options.send_application_alerts.#{@provider.send_application_alerts}.label") %>
+            <% else %>
+              Information unknown
+            <% end %>
+          </p>
+        </dd>
+        <dd class="govuk-summary-list__actions">
+          <%= link_to alerts_provider_ucas_contacts_path(@provider.provider_code), class: "govuk-link", data: { qa: 'send_application_alerts__change' } do %>
+            Change<span class="govuk-visually-hidden"> email alerts for new applications</span>
+          <% end %>
+        </dd>
+      </div>
       <div class="govuk-summary-list__row">
         <dt class="govuk-summary-list__key">
           Email address (or URL) printed in the letter
```

Comment:
There's a couple of bugs here:
The first property should be the value of the preference, ie should you get alerts or not.
The second property is the email they are sent to.

There is also an existing bug that should be fixed. A copy and paste fail from the entry above I think. "Email address..." should say "Who are the alerts sent to?", and that's where we put the information unknown stuff.